app/styles/print.scss

```diff
@@ -0,0 +1,69 @@
+@media print {
+  // Fixes the overflow during printing
+  #subsidyParent {
+    width: auto;
+    height: auto;
+    overflow: visible;
+    flex-direction: column-reverse; // avoid table collapsing with next element
+  }
+  .au-c-app {
+    // overflow: visible;
+    height: auto;
+  }
+
+  .hide-on-print {
+    display: none;
+  }
```

Comment:
Appuniversum already ships this one as `.au-u-hide-on-print`

Reply:
ooooh okay I'll use that one